src/ansys/mechanical/core/embedding/rpc/default_server.py

```diff
@@ -0,0 +1,131 @@
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Remote Procedure Call (RPC) server."""
+
+import fnmatch
+import os
+
+from ansys.mechanical.core.embedding.app import App
+
+from .server import MechanicalEmbeddedServer
+from .utils import remote_method
+
+
+class DefaultServiceMethods:
+    """Default service methods for MechanicalEmbeddedServer."""
+
+    def __init__(self, app: App):
+        """Initialize the DefaultServiceMethods."""
+        self._app = app
+
+    def __repr__(self):
+        """Return the representation of the instance."""
+        return '"ServiceMethods instance"'
+
+    @remote_method
+    def run_python_script(
+        self, script: str, enable_logging=False, log_level="WARNING", progress_interval=2000
+    ):
+        """Run scripts using Internal python engine."""
+        result = self._app.execute_script(script)
+        return result
+
+    @remote_method
+    def run_python_script_from_file(
+        self,
+        file_path: str,
+        enable_logging=False,
+        log_level="WARNING",
+        progress_interval=2000,
+    ):
+        """Run scripts using Internal python engine."""
+        return self._app.execute_script_from_file(file_path)
+
+    @remote_method
+    def clear(self):
+        """Clear the current project."""
+        self._app.new()
+
+    @property
+    @remote_method
+    def project_directory(self):
+        """Get the project directory."""
+        return self._app.ExtAPI.DataModel.Project.ProjectDirectory
+
+    @remote_method
+    def list_files(self):
+        """List all files in the project directory."""
+        list = []
+        mechdbPath = self._app.ExtAPI.DataModel.Project.FilePath
+        if mechdbPath != "":
+            list.append(mechdbPath)
+        rootDir = self._app.ExtAPI.DataModel.Project.ProjectDirectory
+
+        for dirPath, dirNames, fileNames in os.walk(rootDir):
+            for fileName in fileNames:
+                list.append(os.path.join(dirPath, fileName))
+        files_out = "\n".join(list).splitlines()
+        if not files_out:  # pragma: no cover
+            print("No files listed")
+        return files_out
```

Comment:
```suggestion
        file_list = []
        mechdb_path = self._app.ExtAPI.DataModel.Project.FilePath
        if mechdb_path:
            file_list.append(mechdb_path)
        root_dir = self._app.ExtAPI.DataModel.Project.ProjectDirectory

        for root, dir_names, file_names in Path(root_dir).walk():
            for file_name in file_names:
                file_list.append(os.path.join(root, file_name))
        files_out = "\n".join(file_list).splitlines()
        if not files_out:  # pragma: no cover
            print("No files listed")
        return files_out
```
Changed variables to use snake case & used ``Path`` instead of ``os``